projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -3,10 +3,13 @@
   class="cx-card"
   [tabindex]="border ? 0 : -1"
   [attr.role]="content.role ?? null"
-  [attr.aria-label]="
-    content.label ? (content.label | cxTranslate: { number: this.index }) : null
+  [attr.aria-labelledby]="
+    content.header ? 'content-header' + (index >= 0 ? '-' + index : '') : null
+  "
+  [attr.aria-describedby]="
```

Comment:
Because we already have the `ariaDescribedBy()` getter, could we have this logic in there? It might also be good to have `ariaLabelledBy()` to do the same with. We should add comments and tests too as the logic is conditional.

Reply:
I've updated it.